TestApp/app/App.js

```diff
@@ -108,16 +107,13 @@ Crashes.setListener({
 
   onBeforeSending() {
     console.log('Will send crash. onBeforeSending is invoked.');
-    Toast.show('Sending crashes...');
```

Comment:
Any substitution for this..?

Reply:
Logging at least

Reply:
Good point. Replaced `Toast.show` with console logs.